backend/app/services/group_member_helper.py

```
@@ -292,3 +292,266 @@ def count_group_members_by_role(db: Session, group_name: str, role: str) -> int:
+                ResourceMember.entity_type == entity_type,
+                ResourceMember.status == MemberStatus.APPROVED.value,
+            )
+            .all()
+        )
+
+        entity_namespace_ids = [m.resource_id for m in entity_members]
+        entity_namespaces = (
+            (
+                db.query(Namespace)
+                .filter(
+                    Namespace.id.in_(entity_namespace_ids), Namespace.is_active == True
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)